lib/msf/core/payload/windows/x64/meterpreter_loader.rb

```diff
@@ -42,22 +42,23 @@ def asm_invoke_metsrv(opts={})
           push rbp              ; save rbp
           mov rbp, rsp          ; set up a new stack frame
           sub rsp, 32           ; allocate some space for calls.
+          and rsp, ~0xF         ; Ensure RSP is 16 byte aligned 
         ; GetPC
           call $+5              ; relative call to get location
           pop rbx               ; pop return value
         ; Invoke ReflectiveLoader()
           ; add the offset to ReflectiveLoader()
-          add rbx, #{"0x%.8x" % (opts[:rdi_offset] - 0x11)}
+          add rbx, #{"0x%.8x" % (opts[:rdi_offset] - 0x15)}
```

Comment:
I'll dig on my end, will figure this out. Thanks for sanity checking the transports.